packages/gestalt/src/BannerOverlay.js

```diff
@@ -0,0 +1,276 @@
+   */
+  helperLink?: {
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick?: ({
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    }) => void,
+  },
+  /**
+   * Adds an optional primary button for user interaction.
+   * Main action for users to take on BannerOverlay. If href is supplied, the action will serve as a link. See OnLinkNavigationProvider to learn more about link navigation.
+   * If no href is supplied, the action will be a button.
+   * The accessibilityLabel should follow the Accessibility guidelines.
+   * See the Primary action variant to learn more.
+   */
+  primaryAction?:
+    | {
+        accessibilityLabel: string,
+        href: string,
+        label: string,
+        onClick?: $ElementType<ElementConfig<typeof ButtonLink>, 'onClick'>,
+        rel?: $ElementType<ElementConfig<typeof Link>, 'rel'>,
+        role: 'link',
+        size?: $ElementType<ElementConfig<typeof Button>, 'size'>,
+        target?: $ElementType<ElementConfig<typeof Link>, 'target'>,
+      }
+    | {
+        accessibilityLabel: string,
+        label: string,
+        onClick: $ElementType<ElementConfig<typeof Button>, 'onClick'>,
+        role?: 'button',
+        size?: $ElementType<ElementConfig<typeof Button>, 'size'>,
+      },
+  /**
+   * An optional thumbnail to display next to the text.
+   */
+  thumbnail?:
+    | { image: Element<typeof Image> }
+    | { avatar: Element<typeof Avatar> }
+    | { icon: Element<typeof Icon> },
+  /**
+   *  Heading of BannerOverlay. Content should be [localized](https://gestalt.pinterest.systems/web/banneroverlay#Localization). See the [Text variant](https://gestalt.pinterest.systems/web/banneroverlay#Text) to learn more.
+   */
+  title?: string | Element<typeof Text>,
+  /**
+   *  zIndex of BannerOverlay. See the [zIndex guidelines](https://gestalt.pinterest.systems/web/banneroverlay#zIndex) to learn more.
+   */
+  zIndex?: Indexable,
+};
+
+/**
+ * [BannerOverlays](https://gestalt.pinterest.systems/web/banneroverlay) displays short educational messages when users have performed actions that indicate some intent, such as a related pin tap or idea pin swipe. It is scrim-less, meaning users can scroll content underneath without having a wash layer on top of the content.
+ *
+ * BannerOverlay is a sticky component triggered by scroll, on tap or long-press, placed at the bottom of the screen.
+ * ![BannerOverlay light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/BannerOverlay.spec.mjs-snapshots/BannerOverlay-chromium-darwin.png)
+ */
+export default function BannerOverlay({
+  message,
+  title,
+  dismissButton,
+  helperLink,
+  primaryAction,
+  offsetTop = 130,
```

Comment:
i think maybe the defaults are 0? to the engineer to find the correct offset?

Reply:
DONE